### PR TITLE
op-chain-ops: port hh tasks to go

### PR DESCRIPTION
**Pull request by tynes**
**Description**

The hardhat tasks responsible for ensuring that the chain has been completely rolled up during the migration run out of memory. They are written in typescript using hardhat. This ports them to go to prevent them from running out of memory.

### CODE REVIEW

**Comment by tynes** [comment]
needs some linting fixes still

File: op-chain-ops/cmd/rollover/main.go [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+
+				for {
+					shutoffBlock, err := addressManager.GetAddress(&bind.CallOpts{}, "DTL_SHUTOFF_BLOCK")
+					if err != nil {
+						return err
+					}
+					if shutoffBlock.Big().Cmp(common.Big0) != 0 {
+						break
+					}
+					time.Sleep(3 * time.Second)
```
Comment (trianglesphere) [inline op-chain-ops/cmd/rollover/main.go:64]: nit: logging to indicate shutoff block is not set yet.

**Review by trianglesphere — APPROVED** [review]

File: op-chain-ops/cmd/rollover/main.go [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+	if err != nil {
+		return nil, err
+	}
+	return json, nil
+}
+
+// toBlockNumArg is able to handle the conversion between a big.Int and a
+// string blocktag. This function should be used in JSON RPC interactions
+// when fetching data by block number.
+func toBlockNumArg(number *big.Int) string {
```
Comment (trianglesphere) [inline op-chain-ops/cmd/rollover/main.go:247]: Lint says that this is unused

**Comment by tynes** [comment]
Thank you for the feedback @trianglesphere

**Comment by bettimax0** [comment]
Nice

**Pull request merged**